lib/device.js

```diff
@@ -1310,17 +1313,32 @@ const async = require('async'),
             } else if(parseDirPath) {
                 // the path has only "directory" without parseBase
                 // for example, "ares-device -r -s /"
-                options.fileName = createDateFileName(null, csvFormat);
+                options.fileName = createDateFileName(fileNameSeperator, csvFormat);
                 options.destinationPath = path.resolve(parseDirPath);
             }
         }
 
         options.csvPath = path.resolve(options.destinationPath, options.fileName);
-        if (fs.existsSync(options.csvPath)) {
-            fs.unlinkSync(options.csvPath);
-        }
-        log.info("device#makeCSVOutputPath()", "csvPath:", options.csvPath);
-        next();
+        log.verbose("device#makeCSVOutputPath()", "csvPath:", options.csvPath);
+
+        fs.open(options.csvPath, 'w', function(err, fd) {
+            if (err) {
+                return next(errHndl.getErrMsg(err));
+            }
+            fs.close(fd, function(error) {
+                if (error) {
+                    return next(errHndl.getErrMsg(error));
+                }
+                log.verbose("device#makeCSVOutputPath()", options.csvPath + " is closed");
```

Comment:
Why don't you use "silly" instead of "verbose".
It's just a suggestion. If you want to use "verbose", please keep it.

Reply:
I think this is better to keep "verbose". I will not change this.